### PR TITLE
Feature#54 user form design

### LINKED ISSUE #54: add design to user form with picture.
# REQUIREMENTS
Add a design to users' forms.

# Acceptance Criteria
List all required acceptance criteria
- [ ] works correctly.

### PR DESCRIPTION
**Pull request by DerekAyala**
# Description
What did you implement/Why did you implement this:
 - design to user edit form with picture functionality.

  ## Screenshots 
User edit form
Screenshot from 2022-02-25 18-57-25

delete account 
Screenshot from 2022-02-25 18-57-29

profile view
Screenshot from 2022-02-25 19-02-41

### CODE REVIEW

File: app/assets/stylesheets/components/_forms.scss [hunk 1]
```diff
@@ -22,6 +22,13 @@
     box-shadow: 2.5px 2.5px 5px 0px rgba(0,0,0,0.75);
   }
 
+  &__button--delete {
+    background-color: #9C1C1C;
+    padding: 0.5rem 2rem;
+    border-radius: 1.5rem;
+    color: white;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_forms.scss]: Sort this block

File: app/assets/stylesheets/components/_forms.scss [hunk 2]
```diff
@@ -47,6 +54,14 @@
     gap: 0.5rem;
   }
 
+  &__flex {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_forms.scss]: sort this block

File: app/assets/stylesheets/components/_profile.scss [hunk 3]
```diff
@@ -31,6 +31,20 @@
     padding: 1rem 2rem;
   }
 
+  a{
+    text-decoration: none;
+    color: black;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_profile.scss:36]: Sort it and add extra space between label a and your bracket

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by LuisGomez89 — APPROVED** [review]

**Review by IrenehMDev — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**